plugins/aiops/pkg/gateway/modeltraining.go

```diff
@@ -95,7 +97,14 @@ func (p *AIOpsPlugin) GetModelStatus(ctx context.Context, in *emptypb.Empty) (*m
 	}
 	result, err := p.kv.Get().Get("modelTrainingStatus")
 	if err != nil {
-		return nil, status.Errorf(codes.NotFound, "Failed to get model training status from Jetstream: %s", err)
+		if errors.Is(err, nats.ErrKeyNotFound) {
+			return &modeltraining.ModelStatus{
+				Status:     string(msg.Data),
+				Statistics: &modeltraining.ModelTrainingStatistics{},
```

Comment:
I think its okay to pass in an empty statistics object as well. When working earlier, noticed that the admin UI service required a statistics field so that's why I am passing it in here.